decoders/connector/netvox/ra02g/v1.0.0/payload-config.jsonc

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "../../../../../schema/connector_details.json",
+    "description": "../description.md",
+    "install_text": "\nRA02G is a smart indoor sensor designed for high-reliability wireless control and alarm detection. It detects smoke, noise, and tampering, providing versatile alarm functionality over LoRaWAN™.\n\nMore information about this device can be found [here.](http://www.netvox.com.tw/products.asp?pro=RA02G) \n",
+    "install_end_text": "",
+    "device_annotation": "Wireless cigarette smoke, vaping, and anti-tampering alarm sensor with customizable audio alerts and power outage detection.",
+    "device_parameters": [],
+    "networks": [
+      "../../../../network/lorawan-citykinect/v1.0.0/payload.js",
```

Comment:
Just a reminder: Make sure you've added all the networks this connector can be used with.